### DISCUSSION TITLE
backtick not working

### DISCUSSION THREAD

**Discussion by eric248550** [remarkjs/remark#1114] [Q&A] [answered]
In markdown we can use backticks to make `highlight`, but not working in my app
```
`highlight`
```
`highlight`

```javascript
import React from "react";
import ReactMarkdown from 'react-markdown';
import ReactDom from 'react-dom';
// math equation $$
import remarkMath from 'remark-math';
import rehypeKatex from 'rehype-katex';
import 'katex/dist/katex.min.css'
// box, table, 
import remarkGfm from 'remark-gfm';
// html
import rehypeRaw from 'rehype-raw'
// import rehypeHighlight from 'rehype-highlight'

const markdown = `
# A demo of \`react-markdown\`

\`react-markdown\` is a markdown component for React.
`

const app = () => {
  return (
  <ReactMarkdown
    children={markdown}
    remarkPlugins={[remarkGfm, remarkMath]}
    rehypePlugins={[rehypeRaw, rehypeKatex]}
  />
  )
};
```
expected result:
![](https://i.imgur.com/PKLej0c.jpg)

my result
![](https://i.imgur.com/a1S3fFq.jpg)

**Comment by wooorm** [comment, marked as answer]
They work fine.
You just use tailwind or so to add them back.
Fix your css.

**Reply by eric248550** [reply]
How can I do it? I did not find it in the document.

**Reply by ChristianMurphy** [reply]
@eric248550 the example you shared does not have the issue you describe, here is a runnable example of it working in a clean sandbox https://codesandbox.io/s/immutable-leaf-e56grq
The issue you are seeing is one which has been seen by the community before, and comes from your stylesheets, most often tailwind css.
And comes from wherever in your project you've applied stylesheets/tailwind not from the markdown document or `react-markdown`.
